src/components/navbars/DashboardNav.jsx

```diff
@@ -1,14 +1,56 @@
-import DashboardNavStudent from "./DashboardNavStudent";
-import DashboardNavTeacher from "./DashboardNavTeacher";
-import { useAuth } from "../../AuthProvider";
+import Classes from '../../assets/icons/classes.png';
```

Comment:
Well done! Just to keep things tidy let's move the actual data sets to their own module.